src/main/java/duke/exception/DukeIncorrectTaskDescription.java

```diff
@@ -0,0 +1,34 @@
+package duke.exception;
+
+import duke.task.Deadline;
+import duke.task.Event;
+
+/**
+ * Represents the IncorrectTaskDescription Exception class
+ * that is thrown when the user does not enter description
+ * after wanting to add a task according to the format.
+ */
+
+public class DukeIncorrectTaskDescription extends DukeIncorrectInputs {
+    private static final String PREFIX = "To log a";
+    private static final String SUFFIX = ", you have to log the " +
```

Comment:
`+` should be on the next line 

(e.g):
```java
setText("Long line split"
        + "into two parts.");
```

(i've omitted comments for similar issues)